### PR TITLE
docs(argo-cd): Add migration guide to README for Config Management Plugins

### PR DESCRIPTION
**Pull request by kzap**
Fixes: https://github.com/argoproj/argo-helm/issues/2359

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Comment by yu-croco** [comment]
Hi @kzap , thank you for your PR.
Please fix failed CI, otherwise LGTM. 👍

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -26,5 +26,10 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
-    - kind: changed
-      description: Upgrade Argo CD to v2.9.2
+    - kind: added
+      description: [Docs] Added migration steps to README for Config Management Plugins settings available since 5.26.0
+      links:
```
Comment (tico24) [inline charts/argo-cd/Chart.yaml:31]: It'll be these links that are upsetting the CI. I'd just drop them.

Comment (tico24) [reply charts/argo-cd/Chart.yaml:31]: oh and, [docs] either needs the square brackets removed, or put the whole thing in quotes

Comment (kzap) [reply charts/argo-cd/Chart.yaml:31]: thanks, i'll check it again, just added quotes

**Review by tico24 — APPROVED** [review]

File: charts/argo-cd/Chart.yaml (hunk 1 above)
Comment (kzap) [reply charts/argo-cd/Chart.yaml:31]: updated, looks like it was just the `[` and `]` :)

Comment (mkilchhofer) [reply charts/argo-cd/Chart.yaml:31]: Contributors tend to forget updating these fields in the future. We can test it for a while but I'd preffer to not add more artifacthub metadata to a release than documented in our contrib guideline

@jmeridth your opinion?

**Review by yu-croco — APPROVED** [review]

File: charts/argo-cd/Chart.yaml (hunk 1 above)
Comment (mkilchhofer) [reply charts/argo-cd/Chart.yaml]: Thank you @kzap for the update and thank you so much for contibuting the docs update ❤️

**Pull request merged**